### PR TITLE
Adding Trivago logo to streams page

### PR DESCRIPTION
**Pull request by manjuapu**
@guozhangwang Please review

### CODE REVIEW

**Review by miguno — APPROVED** [review]

File: 0110/streams/index.html [hunk 1]
```diff
@@ -17,20 +17,20 @@
...
 <script id="streams-template" type="text/x-handlebars-template">
   <h1>Kafka Streams API</h1>
-     <div class="sub-nav-sticky">
+       <div class="sub-nav-sticky">
       <div class="sticky-top">
         <div style="height:35px">
-          <a  class="active-menu-item" href="/{{version}}/documentation/streams/">Introduction</a>
-          <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+          <a  class="active-menu-item" href="#">Introduction</a>
+          <a href="/{{version}}/documentation/streams/developer-guide">Developers Guide</a>
```
Comment (guozhangwang) [inline 0110/streams/index.html]: This seems a regression to me.

**Review by guozhangwang — COMMENTED** [review]
Left one comment @manjuapu

File: 0110/streams/index.html (hunk 1 above)
Comment (miguno) [reply 0110/streams/index.html]: Oh, good catch.  Yes, should be "Developer", not "Developers".

**Comment by manjuapu** [comment]
@guozhangwang Good catch removed  "s"

**Comment by guozhangwang** [comment]
LGTM. Merged to asf-site.

**Pull request closed without merging**